src/xcode/ENA/ENA/Source/Services/PPAnalyticsCollector/__tests__/Mocks/TestDummyPPAnalyticsDataImplementation.swift

```diff
@@ -10,12 +10,12 @@ final class TestDummyPPAnalyticsDataImplementation: PPAnalyticsData {
 	var lastSubmissionAnalytics: Date?
 	var lastAppReset: Date?
 	var lastSubmittedPPAData: String?
-	var submittedWithQR: Bool = false
 	var currentRiskExposureMetadata: RiskExposureMetadata?
 	var previousRiskExposureMetadata: RiskExposureMetadata?
 	var userMetadata: UserMetadata?
 	var clientMetadata: ClientMetadata?
 	var keySubmissionMetadata: KeySubmissionMetadata?
```

Comment:
```suggestion
	var pcrKeySubmissionMetadata: KeySubmissionMetadata?
```

Reply:
I would do that in a separate renaming PR.